docker/datahub-ingestion/smoke.Dockerfile

```diff
@@ -18,7 +18,5 @@ RUN apt-get update && apt-get install -y \
 RUN DEBIAN_FRONTEND=noninteractive apt-get install -y  openjdk-11-jdk
 
 COPY . /datahub-src
-RUN cd /datahub-src/datahub-ingestion && \
-    sed -i.bak "s/__version__ = \"0.0.0.dev0\"/__version__ = \"$RELEASE_VERSION\"/" src/datahub/__init__.py && \
```

Comment:
Looks like we're loosing this `RELEASE_VERSION` stuff?

I'd like to retain that `sed` command, since it gives us useful information in python telemetry about the versions of docker images that are being used